src/main/groovy/com/github/benmanes/gradle/versions/updates/Resolver.groovy

```diff
@@ -59,14 +59,16 @@ class Resolver {
   final Project project
   final Action<? super ResolutionStrategyWithCurrent> resolutionStrategy
   final boolean checkConstraints
+  final boolean fromBuildSrc
   final ConcurrentMap<ModuleVersionIdentifier, ProjectUrl> projectUrls
 
   Resolver(Project project, Action<? super ResolutionStrategyWithCurrent> resolutionStrategy,
-           boolean checkConstraints) {
+           boolean checkConstraints, boolean fromBuildSrc) {
```

Comment:
The caller could just pass in false for checkConstraints if from buildsrc. Then you don’t need any changes to this class 🙂

Reply:
nice catch, it's done :)